### PR TITLE
Add im7 core GetDelegates() and GetFeatures()

### PR DESCRIPTION
**Pull request by coderkun**
ImageMagick provides a function to list all registered delegates and a second function to list built-in features. Both are very useful for applications to know the capabilities of ImageMagick and to verify an ImageMagick installation.

This Pull Request adds the function `GetDelegates()` which returns the output of `GetMagickDelegates()` and the function `GetFeatures()` which returns the output of `GetMagickFeatures()`.

### CODE REVIEW

File: imagick/magick_version.go [hunk 1]
```diff
@@ -5,11 +5,26 @@
...
 
 /*
+#include <MagickCore/MagickCore.h>
 #include <MagickWand/MagickWand.h>
 */
 import "C"
 import "unsafe"
 
+// Returns the ImageMagick delegates as a string constant.
+func GetDelegates() (delegates string) {
```
Comment (justinfx) [inline imagick/magick_version.go:15]: We don't need to use a named returned value here if we aren't actually using the allocated string.

File: imagick/magick_version.go [hunk 2]
```diff
@@ -5,11 +5,26 @@
...
+	csdelegates := C.GetMagickDelegates()
+
+	return C.GoString(csdelegates)
+}
+
+// Returns the ImageMagick features as a string constant.
+func GetFeatures() (features string) {
```
Comment (justinfx) [inline imagick/magick_version.go:22]: We don't need to use a named returned value here if we aren't actually using the allocated string.

**Review by justinfx — CHANGES_REQUESTED** [review]

File: imagick/magick_version.go (hunk 1 above)
Comment (coderkun) [reply imagick/magick_version.go]: I pushed a fixup commit to change that. Or do you prefer to squash the commits together?

File: imagick/magick_version.go (hunk 2 above)
Comment (coderkun) [reply imagick/magick_version.go]: I pushed a fixup commit to change that. Or do you prefer to squash the commits together?

**Comment by coderkun** [comment]
@justinfx the tests are failing on Linux but it does not seem related to the changes of the Pull Request. Is there anything that I need to do or is the pipeline not working correctlyb

**Comment by justinfx** [comment]
It's a limitation of github actions not being able to use the same env variables in your fork that are set up in the main repo. So the pipeline can pass until it is running on a branch in the main repo. Sucks, but doesn't seem to be another way.

**Comment by coderkun** [comment]
Thanks for merging @justinfx but it looks like the fixup commit did not get squashed into the original commit. Do you want to fix that!

**Comment by justinfx** [comment]
It's not a big deal. Already merged at this point. If it were a lot of fixups I would have wanted to squash it. Thanks!

**Pull request merged**